packages/react-components/components/MoneyAmount.tsx

```diff
@@ -48,4 +50,12 @@ const style = StyleSheet.create({
     fontSize: 24,
     color: colors.darkSecondary,
   },
+  currencyCode: {
+    ...fontStyles.regular,
+    fontSize: 16,
+    lineHeight: 64,
+    color: '#BDBDBD',
```

Comment:
Use the values in colors.ts instead of hard coding please

Reply:
This is a new color used. I'll add it there.